src/components/cylc/tree/TreeItem.vue

```diff
@@ -74,6 +74,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
               v-cylc-object="job.id"
               :key="`${job.id}-summary-${index}`"
               :status="job.node.state"
+              :multiple="node.children.length > 1"
```

Comment:
Works nicely :+1:, quick thought, could we use the status of the previous job for the colour of the "shadow" job?

```suggestion
              :previousStatus="node.children.slice(-2, -1).status"
```

Reply:
Ah! Good idea. This way we avoid having to define yet another color in the theme. And TIL. I didn't know we could go negative in `.slice`, thanks! I think it will be `previousState=node.children[1]` though (the query is sorting reverse, so we display the first item, so the previous is the second I think)

Reply:
Will leave it for @hjoliver and others to review, and apply all suggestions to this PR on Monday 👍 Thanks again!

Reply:
> the query is sorting reverse

Of course 🤦.